arm-web/2016-03-01/ResourceProvider.json

```diff
@@ -207,76 +271,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/publishingCredentials": {
```

Comment:
this would be a breaking change. You are removing the API from the same api-version. When the new SDKs will be published, you would leave the customers high and dry.

/cc @johanste @lmazuel